config/validate/validate_test.go

```diff
@@ -26,6 +26,7 @@ import (
 	json "github.com/ajeddeloh/go-json"
 	"github.com/coreos/ignition/config/validate/astjson"
 	"github.com/coreos/ignition/config/validate/report"
+	"github.com/coreos/ignition/internal/util"
```

Comment:
If you're just grabbing this for `util.BoolToPtr`, these got moved to `config/util`

Reply:
The refactor unfortunately also left using `BoolToPtr` here as an import cycle, so I did a little more shuffling around to fix that too.